libyul/backends/evm/EVMDialect.cpp

```diff
@@ -139,6 +146,17 @@ set<YulString> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 
 map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVersion, bool _objectAccess)
 {
+	auto hasOpCode = [&](evmasm::Instruction _instr, string const& _instrName) -> bool
+	{
+		if (
+			_instr == evmasm::Instruction::PREVRANDAO &&
+			_instrName == "prevrandao"
+		)
+			return _evmVersion.hasPrevRandAO();
```

Comment:
What is this weird capitalisation, `RandAO`?

Reply:
I dunno, I tried my best to make sense of the weird name...

Reply:
It comes from
> A DAO (decentralised autonomous organisation) that anyone can participate in, and the random number is generated by all participants together!

So the names used are RANDAO, randao, Randao -- in this order in the `consensus-specs.` Likely `hasPrevRandao` is the best choice here.